frontend/src/lib/router.tsx

```diff
@@ -155,6 +157,19 @@ const LazyGraphView = React.lazy(() =>
   import('../components/resourceMap/GraphView').then(it => ({ default: it.GraphView }))
 );
 
+/**
+ * Shows a loader whilst it lazy loads the GraphView component.
+ */
+function LazyLoadingGraphView() {
+  const { t } = useTranslation();
+
+  return (
+    <React.Suspense fallback={<Loader title={t('Loading')} />}>
```

Comment:
I'd put this into RouteSwitcher.tsx:56 so it would work for any other lazy loaded route as well